synapse/push/httppusher.py

```diff
@@ -294,8 +294,13 @@ def dispatch_push(self, event, tweaks, badge):
             defer.returnValue([])
         try:
             resp = yield self.http_client.post_json_get_json(self.url, notification_dict)
+            logger.info(
+                "Sent to: %s push-notify: %r get response: %r" % (
+                    self.url, notification_dict, resp
+                )
+            )
         except:
-            logger.warn("Failed to push %s ", self.url)
+            logger.debug("Failed to push %s %r" % (self.url, notification_dict))
```

Comment:
@erikjohnston I believe this is not a good idea - the data will fall into different lines of the file, which makes it difficult to further process them. The essence of the proposed change is that, if necessary, register data in a log in a form convenient for analysis. 
Such a log is necessary first of all for the control of data going to a third-party push gateway server.

Regarding confidentiality: may be better just cut message text from notification dict before send to push gateway? Better yet, is it a customizable option?

I'm not so worried that my server administrator will read my messages, like what push gateway administrator can read them.